Slugbotics/ARM
# Changed the vision/controller system to work off of labels.\n Integrated the controller with the LLM.

This is a super fun PR, the arm can tell you what it can see, and can use natural language to set what it looks at!

To test:
Run the main

Ask the arm what it can see, make sure its correct

Ask the arm to look at something different from what its currently looking at, make sure it does

Ask the arm something random, because it will respond!
